Yoast/javascript
# Add Dutch adjective suffixes to make forms

## Summary

Adds adjective suffixes to regular Dutch adjectives.

## Test instructions

This PR can be tested by following these steps:

* Add more word stems to the spec, together with all of the expected word forms (excluding the stem). Make sure all the expected forms are created.
  * To get stems, run some adjectives to the stemmer to get stems first. E.g., from the input word `korte` you should get the stem `kort`.
  * Use the stem you got in the adjective spec. You should then get forms such as `korte`, `korter`, `kortst` etc., plus also some nonsense forms. (In principle, obtaining nonsense forms is expected behavior. But if you obtain lots of nonsense forms that are ambiguous with other words, this might lead to too many false positives. So also keep an eye on the nonsense forms.)

### Note: 
Please change the package.json premium data specification to `feature/Dutch-morphology` after merging this PR.

## Quality assurance


Fixes #323
